griptape/drivers/structure_run/griptape_cloud_structure_run_driver.py

```diff
@@ -44,7 +44,7 @@ def try_run(self, *args: BaseArtifact) -> BaseArtifact:
         except (exceptions.RequestException, HTTPError) as err:
             return ErrorArtifact(str(err))
 
-    def _get_structure_run_result(self, structure_run_id: str) -> InfoArtifact | TextArtifact | ErrorArtifact:
+    def _get_structure_run_result(self, structure_run_id: str) -> InfoArtifact | BaseArtifact | ErrorArtifact:
```

Comment:
should `InfoArtifact` and `ErrorArtifact` be removed from the type hint?

Reply:
Those are explicitly separate code paths, so I thought it was useful to leave them. I do know they inherit from BaseArtifact though, so *technically* they don't need to be there.

Reply:
That's how I reasoned about it too, I think we should keep them.